packages/cli-kit/src/public/common/object.ts

```diff
@@ -48,13 +49,9 @@ export function pickBy<T, S extends T>(
  * @returns Returns the new mapped object.
  */
 export function mapValues<T extends object, TResult>(
-  obj: T | null | undefined,
+  object: T | null | undefined,
```

Comment:
I prefer to avoid types as variable names, like "string" or "object" to avoid confusion. What about `source` here?

Reply:
Good idea. I'll change it!